firebase.config.js

```diff
@@ -0,0 +1,13 @@
+import { initializeApp } from "firebase/app";
+
+const firebaseConfig = {
```

Comment:
put these keys in .env file and remove the .env from git history by using this command
git rm .env --cached
git commit -m "Any message"
Also add .env in .gitignore file